packages/firestore/CHANGELOG.md

```diff
@@ -1,4 +1,12 @@
-# Unreleased (0.8.0)
+# Unreleased
+
+# 0.8.2
+- [fixed] Fixed an issue where native ES6 module loading was not working.
```

Comment:
Is it worth mentioning webchannel in this at all?

Reply:
I just used the public release notes.